components/country/overview-charts.js

```diff
@@ -0,0 +1,173 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Flex, Box } from 'ooni-components'
+import {
+  VictoryChart,
+  VictoryBar,
+  VictoryStack,
+  VictoryAxis,
+  VictoryTheme,
+  VictoryLine,
+  VictoryVoronoiContainer,
+} from 'victory'
+
+import { testGroups } from '../test-info'
+
+const Circle = styled.span`
+  height: 16px;
+  width: 16px;
+  border-radius: 50%;
+  background-color: ${props => props.color};
+`
+const StyledTestGroupSelector = styled(Flex)`
+  cursor: pointer;
+  &${Box}:hover {
+    text-shadow: 1px 1px 1px black;
+  }
+`
+const TestGroupSelector = ({ testGroup, active, onClick }) => (
+  <StyledTestGroupSelector m={2} onClick={() => onClick(testGroup)}>
+    <Circle color={active ? testGroups[testGroup].color : '#ced4da'} />
+    <Box mx={1} color={!active && '#ced4da' }> {testGroups[testGroup].name} </Box>
```

Comment:
What is the color code #ced4da for?

Reply:
It is basically `gray4` from the theme. And it is to gray out test groups not included in the charts below.

Reply:
Will change it to `theme.colors.gray4`